doc/build-macos.md

```diff
@@ -1,57 +1,54 @@
-Mac OS X Build Instructions and Notes
-====================================
-The commands in this guide should be executed in a Terminal application.
-The built-in one is located in `/Applications/Utilities/Terminal.app`.
+MacOS Build and Run Instructions
 
-Preparation
------------
-Install the OS X command line tools:
+The commands in this guide should be executed in a Terminal application. The built-in one is located in /Applications/Utilities/Terminal.app.
 
-`xcode-select --install`
+Preparation:
 
-When the popup appears, click `Install`.
+Install the MacOS command line tools:
 
-Then install [Homebrew](https://brew.sh).
+xcode-select --install
 
-Dependencies
-----------------------
+When the popup appears, click Install.
 
-    brew install automake berkeley-db4 libtool boost --c++11 miniupnpc openssl pkg-config qt
+Then install Homebrew.
 
-If you want to build the disk image with `make deploy` (.dmg / optional), you need RSVG
+Install Dependencies:
 
-    brew install librsvg
+brew install automake berkeley-db4 libtool boost --c++11 miniupnpc openssl pkg-config qt
+If you want to build the disk image with make deploy (.dmg / optional), you need RSVG
 
-NOTE: Building with Qt4 is still supported, however, could result in a broken UI. Building with Qt5 is recommended.
+brew install librsvg 	(to install RSVG)
 
-Build Gridcoin
-------------------------
 
-1. Clone the Gridcoin source code and cd into `Gridcoin-Research`
+To Build Gridcoin:
 
-        git clone https://github.com/gridcoin/Gridcoin-Research
-        cd Gridcoin-Research
+Clone the Gridcoin source code and cd into Gridcoin-Research with;
 
-2.  Build Gridcoin:
+ git clone https://github.com/gridcoin/Gridcoin-Research
+ cd Gridcoin-Research
 
-    Configure and build the headless gridcoin binaries as well as the GUI (if Qt is found).
 
-    You can disable the GUI build by passing `--without-gui` to configure.
+Configure and build the headless gridcoin binaries as well as the GUI (if Qt is found).
 
-        ./autogen.sh
-        ./configure
-	make
+You can disable the GUI build by passing --without-gui to configure.
 
-    The daemon binary is placed in src/ and the gui client is found in src/qt/ . Run the gui client for testnet for example with
-    	
-	./gridcoinresearch -testnet
+./autogen.sh
+./configure
+make
 
-3.  It is recommended to build and run the unit tests:
+make clean 	[cleans out previous builds!!!!!! Do this between versions]
```

Comment:
Make clean needs to be run before make. Alternative formulation:

If you are building a new version, you might want to `make clean` first.